src/Symfony/Bundle/FrameworkBundle/Command/DebugAutowiringCommand.php

```diff
@@ -15,6 +15,8 @@
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\DependencyInjection\Debug\AutowiringInfoManager;
+use Symfony\Component\DependencyInjection\Debug\AutowiringTypeInfo;
```

Comment:
looks like this isn't used.

Reply:
They are both used - one I think just for phpdoc :). We have a PR bot (fabbot) that validates cs, including unused use statements

Reply:
Oh how come include namespaces for just docblocks? Interesting!

Reply:
It lets you use the short class names in phpdoc - like @param Foo - editors will use the use statement to know the full class name that you’re referencing :)